### ISSUE TITLE
RateLimit

### ISSUE THREAD

**Issue by germs12** [timcase/echowrap#10]
I've been trying to find a way to determine the current rate limit while using the echowrap gem.  Unfortunately I can't find any documentation or examples that show how to find it.  Is this a feature echowrap offers?  Thanks and great work @timcase

**Comment by timcase** [comment]
No, there isn't a good way to do this right now. The rate limit is returned in every api call in the response headers: http://developer.echonest.com/docs/v4. This will be something that I'll address at some point, but at the moment I'm not sure when I'll get to it. A workaround might be to do what they suggested and use NET/HTTP to perform an API call and then inspect the headers, sorry I don't have a better answer...
